static/css/main.ie.css

```diff
@@ -1039,7 +1039,7 @@ input.btn::-moz-focus-inner {
 
 .btn__disabled:focus,
 .btn__disabled.focus {
-  outline-color: #75787b;
+  outline-color: #e3e4e5;
```

Comment:
Hmm, I'll check into it. This is the override file coming from OAH, which may be more current.